spec/index.bs

```diff
@@ -7140,9 +7146,16 @@ run the following steps:
                 1. Set |storageTextureLayout|.{{GPUStorageTextureBindingLayout/format}} to |resource|'s format.
                 1. Set |storageTextureLayout|.{{GPUStorageTextureBindingLayout/viewDimension}} to |resource|'s dimension.
 
-                1. If |resource| is for a write-only storage texture:
-
-                    1. Set |storageTextureLayout|.{{GPUStorageTextureBindingLayout/access}} to {{GPUStorageTextureAccess/"write-only"}}.
+                1. If the access mode is:
+                   
+                    <dl class=switch>
+                        : `read`
+                        :: Set |textureLayout|.{{GPUStorageTextureBindingLayout/access}} to {{GPUStorageTextureAccess/"read-only"}}
+                        : `write`
+                        :: Set |textureLayout|.{{GPUStorageTextureBindingLayout/access}} to {{GPUStorageTextureAccess/"write-only"}}
```

Comment:
Could we have text to automatically upgrade write-only to read-write if needed (even if it is not usable in the API yet?)

Reply:
Do you mean have the API "read-write" be compatible with WGSL "write" and use that if the format works?

Reply:
When `rw-storage-texture-tier-1` is introduced, you could merge bindings on some devices, but not others. Is that desirable?

Reply:
Yes, that seems ok, because the optional features are opt-in.

Reply:
I'm ok with doing this. Do we need larger group agreement first?

Reply:
We can put it as tacit resolution.

Reply:
PTAL at those changes.